src/main.rs

```diff
@@ -204,8 +207,14 @@ fn run() -> Result<()> {
                                  .value_name("COMMIT-MESSAGE")
                                  .help("Commit message that will be used on import")
                                  .default_value("cobalt site import")
-                                 .takes_value(true)))
-        .get_matches();
+                                 .takes_value(true)));
+
+    #[cfg(all(feature="syntax-highlight"))]
+    let app_cli =
+        app_cli
+            .subcommand(SubCommand::with_name("list-syntax-themes").about("list available themes"));
```

Comment:
I think its a matter of perspective. Is syntax highlighting optional because people want to turn it off or because it is platform specific?

I don't know the implementer's intent and so I lean towards platform specific. The benefit to it being a dedicated feature are platforms or environments where the external dep isn't as easily available and to make the conditional nature of this feature more explicit.